### PR TITLE
Update Slider to reflect changes immediately

### PR DESCRIPTION
**Pull request by dratwas**
Fixes https://github.com/wordpress-mobile/gutenberg-mobile/issues/2192
Gutenberg PR: https://github.com/WordPress/gutenberg/pull/22421

To test:
1.
- open mobile app
- add spacer block
- open settings of the spacer block
- change the height of spacer - change should be visible while moving the slider
2.
- add button block
- open settings of that button
- change the border-radius - change should be visible while moving the slider

PR submission checklist:

- [x] I have considered adding unit tests where possible.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: RELEASE-NOTES.txt [hunk 1]
```diff
@@ -13,6 +13,7 @@
 * [**] Tooltip for page template selection buttons
 * [*] Fix button alignment in page templates and make strings consistent
 * [*] Add support for displaying radial gradients in Buttons and Cover blocks
+* [**] Reflect changes of slider in block settings immediately
```
Comment (geriux) [inline RELEASE-NOTES.txt]: This should go for `1.30.0`

**Comment by mchowning** [comment]
We're preparing to cut the 1.30 release branches now. Let's merge this ASAP or update the milestone to 1.31.

**Comment by dratwas** [comment]
Hey @mchowning, unfortunately, we had to revert the PR that fixed the issue with creating undo levels when using the slider. https://github.com/WordPress/gutenberg/pull/22933

The issue is described here https://github.com/wordpress-mobile/gutenberg-mobile/issues/2192#issuecomment-630010164

I'm moving it to the milestone 1.31

**Comment by mchowning** [comment]
👋  @dratwas! We're close to merging the monorepo changes 🎉 , and we will be cutting the release branches as soon as the monorepo changes are merged to gutenberg-mobile. As a result, we may be cutting the 1.32 release branches earlier than normal.

If these changes can wait until 1.32, please bump this to the 1.32 milestone and wait to merge this until after the release is cut. That way we don't add more changes to the 1.31 release than are necessary—this release is already going to have a lot more changes than a normal release because of the monorepo changes. If it is important that this PR make the 1.31 release, however, please try to merge this ASAP so it is merged before we cut the release. 🙇

**Comment by mchowning** [comment]
👋 @dratwas  ! We're getting ready to cut the 1.32 release branch, so I'm bumping this to the 1.33 milestone. If this needs to make the 1.32 release though, go ahead and change the milestone back and try to get this merged as soon as possible. If you're trying to get this into the release, please also let us know in Slack. :bow:

**Comment by dratwas** [comment]
Hey @mchowning thanks for the heads up i need to wait for the https://github.com/wordpress-mobile/WordPress-Android/pull/12144 with this one.

**Comment by SergioEstevao** [comment]
👋 @dratwas ! We're getting ready to cut the 1.33 release branch, so I'm bumping this to the 1.34 milestone. If this needs to make the 1.33 release though, go ahead and change the milestone back and try to get this merged as soon as possible. If you're trying to get this into the release, please also let us know in Slack. 🙇

**Comment by ceyhun** [comment]
👋 @dratwas ! We're getting ready to cut the 1.34 release branch, so I'm bumping this to the 1.35 milestone. If this needs to make the 1.35 release though, go ahead and change the milestone back and try to get this merged as soon as possible. If you're trying to get this into the release, please also let us know in Slack. 🙇

**Comment by ceyhun** [comment]
@dratwas heads up we'll be cutting the 1.35 release branch beginning of next week. Please try to get this merged by than or bump the milestone 🙇

**Comment by cameronvoell** [comment]
> @dratwas heads up we'll be cutting the 1.35 release branch beginning of next week. Please try to get this merged by than or bump the milestone 🙇

Bumping milestone

File: RELEASE-NOTES.txt [hunk 2]
```diff
@@ -1,6 +1,7 @@
 1.35.0
 ------
 * Creating undo levels less frequently.
+* Reflect changes of slider in block settings immediately.
```
Comment (geriux) [inline RELEASE-NOTES.txt]: Shouldn't this be [here](https://github.com/WordPress/gutenberg/blob/master/packages/react-native-editor/CHANGELOG.md) instead?

**Comment by cameronvoell** [comment]
@dratwas not sure if this PR is needed for package.json updates, or if we can close it since I believe the gutenberg side is already merged and updated in gb-mobile develop. If you want to merge this one, I think merging in develop should fix CI (fix PR - https://github.com/wordpress-mobile/gutenberg-mobile/pull/2571). 

I'll cut the branches for 1.36 later in the day on Monday 8/31, see please merge before then if needed, thanks!

**Comment by dratwas** [comment]
Closing since these changes are merged already.

**Pull request closed without merging**